index.test.js

```diff
@@ -0,0 +1,63 @@
+import { addString } from "./index";
+import { toNumber } from "./index";
+import { filterOrange } from "./index";
+
+describe("addString", () => {
+    test('"a" + "b" to equal "ab"', () => {
+        expect(addString("a", "b")).toBe("ab");
+    });
+
+    test('"a" + 1 to equal "a1"', () => {
+      expect(addString("a", 1)).toBe("a1");
+    });
+
+    test('1 + 1 to equal -1', () => {
+        expect(addString(1, 1)).toBe(-1);
+      });
+})
+
+describe("toNumber", () => {
+    test('"123" to equal 123', () => {
+        expect(toNumber("123")).toBe(123);
+    });
+
+    test('123 to equal 123', () => {
+      expect(toNumber(123)).toBe(123);
+    });
+
+    test('"3b3b3b" to equal 3', () => {
+        expect(toNumber("3b3b3b")).toBe(3);
+    });
+
+    test('"b3b3b3" to equal NaN', () => {
+        expect(toNumber("b3b3b3")).toBe(NaN);
+    });
+
+    test('Numbers with a positive sign to be unsigned', () => {
+        expect(toNumber("+5")).toBe(5);
+    });
+
+    test('floating point numbers to be integer', () => {
+        expect(toNumber("-5.9")).toBe(-5);
+    });
+})
+
+describe("filterOrange", () => {
+    test('Array containing "orange"', () => {
+        const fruit = ["orange", "apple", "peach"];
+        expect(filterOrange(fruit)).toEqual(["orange"]);
+    });
+    test('Array not containing orange', () => {
+        const fruit = ["apple", "peach"];
+        expect(filterOrange(fruit)).toEqual([]);
+    });
+
+    test('throws on not array', () => {
+        const fruit = "orange"
+        expect( () => filterOrange(fruit)).toThrow();
+    });
+
+    test('throws on undefined', () => {
+        expect( () => filterOrange(fruit)).toThrow();
+    });
```

Comment:
```suggestion
  test('throws on not array', () => {
        const fruit = "orange"
        const expectErrorFunction = () => {
          filterOrange(fruit);
        }
        expect(expectErrorFunction).toThrow("arr.filter is not a function");
    });

    test('throws on undefined', () => {
      const expectErrorFunction = () => {
        filterOrange(undefined);
      }
      expect(expectErrorFunction).toThrow("Cannot read property 'filter' of undefined");
    });
```

Reply:
Thanks for your review.
I have corrected what you pointed out : https://github.com/matsuwork/jest/pull/1/commits/96633241f370fe57bd4b64d17f230ef31565c9eb
So, I will merge them.
As for the explanatory variables, I will use it as a reference next time.